example-app/SantokuApp/src/navigation/RootStackNav.tsx

```diff
@@ -2,13 +2,27 @@ import {createStackNavigator} from '@react-navigation/stack';
 import React from 'react';
 import {Home, Instructions, ApplicationInfo} from 'screens';
 
+import {ErrorCase} from '../screens/error/ErrorCase';
```

Comment:
`demo`ディレクトリの中にまとめて置いときたいかな。

```suggestion
import {ErrorCase} from '../screens/demo/error/ErrorCase';
```

Reply:
`demo`に移動しましたー

e7010059612a4c7d7c9abb8dda5c50e6cc493d91

Reply:
あー、master見るとdemoディレクトリに画面とか入ってるのか。
ちょっとそれをマージするので確認お待ちください。

Reply:
移動しましたー